public/components/DataSourceAlertsCard/DataSourceAlertsCard.tsx

```diff
@@ -80,7 +89,7 @@ export const DataSourceAlertsCard: React.FC<DataSourceAlertsCardProps> =  ({ get
             <EuiText size="s" color="subdued">Monitor:</EuiText>
           </EuiFlexItem>
           <EuiFlexItem grow={false} >
-            <EuiText size="m">{getTruncatedText(monitorName)}</EuiText>
+            <EuiText size="m">{monitorName}</EuiText>
```

Comment:
how will this impact in case the name is too long?

Reply:
Updated the EuiPanel so that the overflow is hidden. This is what it would look like if the monitor name is too long

![Screenshot 2024-09-19 at 2 10 24 PM](https://github.com/user-attachments/assets/89df1182-4d3b-4fec-9e9a-eacb6bc4c02f)

Reply:
but this doesn't indicate there is more to the name? is there some tooltip to it? Would be better to have some truncation UI

Reply:
Spoke to @xeniatup and decided to dynamically truncate the name according to the card width. Here is what it would look like on two different window sizes

![Screenshot 2024-09-19 at 5 00 42 PM](https://github.com/user-attachments/assets/942c6762-2f2a-45f8-88c3-a86e1b499f1c)

![Screenshot 2024-09-19 at 5 01 14 PM](https://github.com/user-attachments/assets/5e31289a-ede5-4625-93a8-6e04c8b95fe2)